gulpfile.js

```diff
@@ -39,10 +39,7 @@ var whiteListNames = [
   'deepFreezeAndThrowOnMutationInDev',
   'flattenStyle',
   'InitializeJavaScriptAppEngine',
-  'InteractionManager',
   'JSTimersExecution',
-  'merge',
```

Comment:
InteractionManager and merge are already removed dependencies so I'll just clean this up while I'm in here.